make/modules/java.xml/Copy.gmk

```diff
@@ -24,14 +24,18 @@
 #
 
 include CopyCommon.gmk
+include Modules.gmk
 
 ################################################################################
+#
+# Copy property files from share/conf to CONF_DST_DIR LIB_DST_DIR
+#
+JAXPPROPFILE_SRC_DIR := $(TOPDIR)/src/java.xml/share/conf
+JAXPPROPFILE_SRCS := $(wildcard $(JAXPPROPFILE_SRC_DIR)/jaxp*.properties*)
+JAXPPROPFILE_TARGET_FILES := $(subst $(JAXPPROPFILE_SRC_DIR),$(CONF_DST_DIR),$(JAXPPROPFILE_SRCS))
 
-XML_LIB_SRC := $(TOPDIR)/src/java.xml/share/conf
-
-$(CONF_DST_DIR)/jaxp.properties: $(XML_LIB_SRC)/jaxp.properties
+$(CONF_DST_DIR)/%: $(JAXPPROPFILE_SRC_DIR)/%
```

Comment:
The make file changes to copy the properties files look okay but I'm curious about why the naming changes from "XML" to "JAXPPROFILE".

Reply:
If we are changing this file, we should modernize it.

```
$(eval $(call SetupCopyFiles, COPY_XML_MODULE_CONF, \
    DEST := $(CONF_DST_DIR), \
    FILES := $(wildcard $(TOPDIR)/src/java.xml/share/conf/jaxp*.properties*), \
))

TARGETS += $(COPY_XML_MODULE_CONF)
```